src/sprints/sprints.service.ts

```diff
@@ -263,7 +271,23 @@ export class SprintsService {
             },
         });
     }
+    async helper(meetingId: number) {
```

Comment:
I think we should name this something more descriptive then `helper`

Reply:
Actually we don't need this function , I created this to test something but forgot to remove it 😅